connector/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/v2/V2JDBCTest.scala

```diff
@@ -980,4 +980,86 @@ private[v2] trait V2JDBCTest extends SharedSparkSession with DockerIntegrationFu
       )
     }
   }
+
+  def testDatetime(): Unit = {
```

Comment:
can we check how we use `testUpdateColumnNullability`, `testRenameColumn`, etc.? Seems it's different with this approach here.

Reply:
You means remove the call of test?